src/Database/Retry/ReconnectStrategy.php

```diff
@@ -110,7 +110,10 @@ protected function reconnect(): bool
 
         try {
             $this->connection->connect();
-            $this->connection->log('[RECONNECT]');
+            $this->connection->getDriver()->getLogger()?->debug(
```

Comment:
This would be shorter as `$this->connection->getDriver()->logQuery('connection={connection} [RECONNECT]', [...]);`